lib/constants/category.ts

```diff
@@ -8,6 +8,7 @@ const Categories = [
   'ci',
   'custom',
   'dart',
+  'devbox',
```

Comment:
```suggestion
```

It does not need its own category, otherwise category concept is meaningless if we make it 1:1